RF24.h

```diff
@@ -598,6 +598,36 @@ class RF24 {
      * a `1` means the feature is enabled.
      */
     void printPrettyDetails(void);
+    
+    /**
+     * Put a giant block of debugging information in a char array. This function
+     * differs from printDetails() because it makes the information more
+     * understandable without having to look up the datasheet or convert
+     * hexadecimal to binary. Only use this function if your application can
+     * spare extra bytes of memory.
+     *
+     * @warning use a buffer of sufficient size, start with a sizeof >= 1024
+     * 
+     * @code
+     * char buffer[1024] = {'\0};
+     * int _len = 0;
+     * RF24 radio(RF24_RADIO_CE_PIN, RF24_RADIO_CS_PIN); 
+     * setup(){
+     * radio.sprintfPrettyDetails(buffer);
+     * _len = strlen(buffer);
+     * Serial.begin(115200);
+     * Serial.print(buffer);
+     * Serial.print(F("strlen = "));
+     * Serial.println(strlen(buffer));
+     * }
```

Comment:
This sample code block needs proper (4-space) indentation.

Since you copied the docs from the original `printPrettyDetails()`, you should help explain to users how it differs from that (maybe include a link to a C++ reference about using `sprintf()`).

Reply:
I fixed the indentation, and added documentation